src/doc/src/guide/creating-a-new-workspace.md

```diff
@@ -0,0 +1,75 @@
+# Creating a New Workspace
+
+A [workspace][def-workspace] is a collection of one or more packages, called workspace members, that are managed together.
+
+In this chapter we implement the creation of a workspace `new_workspace` from scratch, in which two members `foo-bin` and `bar-lib` are included.
+
+As mentioned in [[workspace] section](../reference/workspaces.md#the-workspace-section), the workspace must have at least one member, either the root package or a virtual manifest. Here we recommend setting the root of the workspace to be a virtual manifest. The reason is as follows:
+
+- The crate namespace at the Cargo level is flat. The tree layout creates another hierarchy and increases the possibility of inconsistencies. 
+- With a flat structure, adding or splitting crates is very easy. 
+- Even larger flattened lists are easier to see at a glance and easier to maintain than smaller tree structures.
+
+Let's look at how a workspace containing a virtual inventory should be created.
```

Comment:
I don't think we used "virtual inventory" elsewhere. Avoid introducing new jargon and keep the guide written in simple English.

```suggestion
Let's move on and create a virtual workspace.
```